CVEs/scripts/scan_vuln.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+images_with_tags=$(cat images.txt)
+
+echo "" > scan_results.txt
+
+for line in $images_with_tags; do
+  trivy image --platform=linux/amd64  --no-progress --output scan.tmp --format json --severity CRITICAL "$line"
```

Comment:
not sure if the `--platform` is needed, you commented it out in the other PR #235